.github/mergify.yml

```diff
@@ -0,0 +1,42 @@
+pull_request_rules:
+  - name: Approve and merge non-major version dependabot upgrades
+    conditions:
+      - author~=^dependabot(|-preview)\[bot\]$
+      - check-success~=lint
+      - title~=^build\(deps[^)]*\). bump [^\s]+ from ([\d]+)\..+ to \1\.
+    actions:
+      review:
+        type: APPROVE
+      merge:
+        method: squash
+
+  - name: Approve and merge Snyk.io upgrades
+    conditions:
+      - author=snyk-bot
+      - check-success~=lint
+      - title~=^\[Snyk\]
+    actions:
+      review:
+        type: APPROVE
+      merge:
+        method: squash
+
+  - name: Close stale PRs (>1 month since last activity)
```

Comment:
Let's hold off on this one for now while we clean up the PR/issues

Reply:
unsure if you're referring to just this rule (closing stale PRs) or mergify in general. 

if it's just the rule, i can split the PR and make this rule its separate PR so that we can get mergify in first and the stale PR rule can be merged when the clean up is done!

Reply:
Just the rule, as discussed earlier! :)

Reply:
done [here](https://github.com/isomerpages/isomercms-frontend/pull/745/commits/0ce96f8ab1f05810f23a07ed4976694099257f84)